inlong-sort/sort-connectors/base/src/main/java/org/apache/inlong/sort/base/Constants.java

```diff
@@ -163,4 +163,11 @@ public final class Constants {
                     .booleanType()
                     .defaultValue(true)
                     .withDescription("Whether ignore the single table erros when multiple sink writing scenario.");
+
+    public static final ConfigOption<Boolean> SINK_MULTIPLE_TYPE_MAP_COMPATIBLE_WITH_SPARK =
+            ConfigOptions.key("sink.multiple.typemap-compatible-with-spark")
+                    .booleanType()
+                    .defaultValue(false)
+                    .withDescription("Because spark do not support iceberg data type: `timestamp without time zone` and"
+                            + "`time`, so type conversions must be mapped to types supported by spark.");
```

Comment:
maped to string ?

Reply:
I guess it's nessasary to mention the conversion explicitly like datetime to varchar